terminus/models/road.py

```diff
@@ -54,24 +54,16 @@ def template(cls):
 
 
 class Street(Road):
-    def __init__(self, name=None):
-        super(Street, self).__init__(self.default_width(), name)
-
-    @classmethod
-    def default_width(cls):
-        return 5
+    def __init__(self, width=5, name=None):
+        super(Street, self).__init__(width, name)
 
     def material_name(self):
         return 'Gazebo/Residential'
 
 
 class Trunk(Road):
-    def __init__(self, name=None):
-        super(Trunk, self).__init__(self.default_width(), name)
-
-    @classmethod
-    def default_width(cls):
-        return 10
+    def __init__(self, width=10, name=None):
```

Comment:
Just to be sure, have you checked that this interface change doesn't break other builders?